### DISCUSSION TITLE
Optimize Top Level Examples Page Content

### DISCUSSION THREAD

**Discussion by Scienfitz** [emdgroup/baybe#65] [Documentation]
Currently, the user gets a hammer thrown at their face when they simply click on examples:
![image](https://github.com/emdgroup/baybe/assets/17951239/c259901f-35d8-45da-ae29-891e3453718f)

This shows all topics, sub topis and subsub topics. Even though a nice grouping is done on the tab bar left, the main page here can be quite offputting.

Suggestion:
Just show the high level topics and maybe ONE sub-level, but not levels even below that. Eg.
Old:
- Backtesting
   - Custom Analytical
      - Necessary Imports ...
      - ...
   - Impute Mode
      - ...
   - ...
- ...

New:
- Backtesting
   - Custom Analytical
   - Impute Mode
   - ...
- ...

This principle is probably best even applied to all pages and not just examples.
@AVHopp

**Comment by AVHopp** [comment]
I guess that this is configurable. Would put this in a PR together with the ordering of the examples mentioned in the other discussion if @AdrianSosic agrees with your proposal here.

**Reply by AdrianSosic** [reply]
Absolutely! I think that probably even just the highest level might suffice here. But if the second level is not too overwhelming, then that should also be OK

**Comment by Scienfitz** [comment]
closing as resolved / implemented
